alarm-frontend/src/Components/Alarm.js

```diff
@@ -1,63 +1,67 @@
-import React, { useContext } from "react";
-import { minutesNumber, hourNumber } from "../fixNumbers";
-import useOption from "../Hooks/useOption";
+import React, { useContext, useEffect, useState } from "react";
 import { Context } from "./Context";
+import '../css/alarm.css';
+import Sound from "../LostWoods.mp3";
+import axios from "axios";
+const BASE_URL = "http://127.0.0.1:5000"
```

Comment:
Shouldn't have to declare this twice! I would pull this out into a constants.js and share in places. Also related to https://github.com/TJKruger1319/alarm-app/pull/3/files#r1298522125